liubai-frontends/liubai-web/.env.template

```
@@ -1,10 +1,13 @@
 
 # API domain
-VITE_API_DOMAIN=     # like https://api.example.com
+VITE_API_DOMAIN=     # like https://api.example.com/
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)